packages/sshnoports/bin/sshnp.dart

```diff
@@ -21,6 +21,7 @@ void main(List<String> args) async {
   try {
     await sshnp.init();
     await sshnp.run();
+    exit(0);
```

Comment:
No, sshnp needs to exit

Reply:
It will still keep the background isolate running which is what we want, but if it doesn't exit, then it won't exit automatically until that background isolate has died on its own.